src/BMI.ts

```diff
@@ -25,6 +28,15 @@ function getBMIState(BMI: number): BMIStatus {
   // Don't change this variable
   let BMIState: BMIStatus = "obese";
   // Write your code here 👇🏼
+  if (BMI<18.5){
+    return BMIState = "underweight";
+  } 
+  else if (BMI >= 18.5 && BMI <25){
```

Comment:
If we reach this else-if then we already **know** that the BMI >= 18.5. We don't need to check again.